src/components/BorrowLimit.js

```diff
@@ -0,0 +1,26 @@
+import React, {Component} from "react";
+import {numm} from "../lib/Utils";
+
+export default class BorrowLimit extends Component {
+
+
+    render() {
+
+        const {userInfo} = this.props;
+
+        return (
+            <div className="borrow-limit">
+                <h3>Borrow Limit</h3>
+                <div className="limit-bar">
+                    <label>0 DAI</label>
```

Comment:
after talking with the designer, the label here should be `userInfo.bCdpInfo.daiDebt`.
note that all over, beside in user input, we only want to present 2 decimals for DAI, and 3 for ETH.